x-pack/plugins/reporting/server/config/create_config.ts

```diff
@@ -6,7 +6,7 @@
 
 import { i18n } from '@kbn/i18n/';
 import crypto from 'crypto';
-import { capitalize } from 'lodash';
+import { upperFirst } from 'lodash';
```

Comment:
Just want to verify this change as it might be breaking. `upperFirst` and `capitalize` have different implementations. I can't recall every place downstream we use this, but it could cause issues since `capitalize` lowercases all subsequent letters.

Reply:
Looks like we're just logging this value, so we're good to go from here!

Reply:
I've tested this while doing the upgrade. From lodash@3 to lodash@4 I think the correct equivalent is `upperFirst`.
If you check here on that example https://jsbin.com/runimajetu/edit?html,js,console,output `upperFirst` is the only one that provides the same output between lodash@3 and lodash@4.